dali/pipeline/operators/op_spec_test.cc

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include <gtest/gtest.h>
+
+#include "dali/pipeline/pipeline.h"
+
+#include "dali/core/common.h"
+#include "dali/pipeline/data/backend.h"
+#include "dali/pipeline/data/buffer.h"
+#include "dali/pipeline/data/tensor.h"
+#include "dali/pipeline/operators/operator.h"
+#include "dali/pipeline/workspace/workspace.h"
+#include "dali/test/dali_test.h"
+
+
+namespace dali {
+
+class SupportGeneratingOp : public Operator<SupportBackend> {
```

Comment:
I think the word "test" should appear somewhere in the name - after all, this operator is globally visible to all operators in dali_test and it'd be nice if it was clearly distinguishable from normal operators exposed by dali.so.

Reply:
Suggested alternatives:
`TestSupportOp_Generator`
`TestSupportOp_Consumer`

Reply:
Done